Header header
float32 length_cm
